sway/commands/move.c

```diff
@@ -0,0 +1,181 @@
+#include <string.h>
+#include <strings.h>
+#include <wlr/types/wlr_output.h>
+#include <wlr/types/wlr_output_layout.h>
+#include <wlr/util/log.h>
+#include "sway/commands.h"
+#include "sway/input/seat.h"
+#include "sway/output.h"
+#include "sway/tree/container.h"
+#include "sway/tree/layout.h"
+#include "sway/tree/workspace.h"
+#include "stringop.h"
+#include "list.h"
+
+static const char* expected_syntax = 
+	"Expected 'move <left|right|up|down|next|prev|first> <[px] px>' or "
```

Comment:
`next`, `prev` and `first` seem to have been removed

Reply:
Good catch